test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -926,5 +930,102 @@ public void VerifyExpectedAndActualBooleanOptionsDictionary(Dictionary<string, D
                 Assert.That(actualValue, Is.EqualTo(expectedValue), $"Actual Property from Service is not equal to default property for {optionRow.Key}");
             }
         }
+
+        /// <summary>
+        /// Verify the generate Tsql model operation
+        /// </summary>
+        [Test]
+        public void GenerateTSqlModelFromSqlFiles()
+        {
+            DacFxService service = new DacFxService();
+            Directory.CreateDirectory(TSqlModelTestFolder);
```

Comment:
I guess this is ok since we're already doing it everywhere, but we should really be trying to use Mocks and dependency injection for stuff like this so we don't have to be writing and reading from disk (which may fail on its own and also ends up leaving a bunch of random stuff laying around on the users machine)